pages/common/ffmpeg.md

```diff
@@ -4,16 +4,20 @@
 
 - Extract the sound from a video and save it as MP3:
 
-`ffmpeg -i {{video_filename}} -vn -ar 44100 -ac 2 -ab 192 -f mp3 {{sound.mp3}}`
+`ffmpeg -i {{path/to/video}} -vn -ar 44100 -ac 2 -ab 192 -f mp3 {{sound}}.mp3`
 
 - Convert frames from a video or GIF into individual numbered images:
 
-`ffmpeg -i {{video_or_gif_filename}} {{image%d.png}}`
+`ffmpeg -i {{path/to/video_or_gif}} {{image%d.png}}`
 
 - Combine numbered images (image1.jpg, image2.jpg, etc) into a video or GIF:
 
-`ffmpeg -f image2 -i {{image%d.jpg}} {{video.mpg_or_video.gif}}`
+`ffmpeg -i {{path/to/image%d.jpg}} -f image2 {{video.mpg_or_video.gif}}`
+
+- Extract a single frame from a video at time mm:ss and save it as a 128x128 resolution image:
```

Comment:
Very cool example! :smiley_cat:

Reply:
I know ! I just used it at work. The first thing I did was check `tldr ffmpeg` and saw this wasn't there. Hence this PR. 😄